lib/src/alpha_picker.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+
+import 'slider_picker.dart';
+
+class AlphaPicker extends StatefulWidget {
+  const AlphaPicker({
+    required this.alpha,
+    required this.onChanged,
+    Key? key,
+  }) : super(key: key);
+
+  final int alpha;
+  final ValueChanged<int> onChanged;
+
+  @override
+  State<StatefulWidget> createState() => _AlphaPickerState();
+}
+
+class _AlphaPickerState extends State<AlphaPicker> {
+  void valueOnChanged(double ratio) {
+    super.widget.onChanged(
+          ratio.toInt(),
```

Comment:
Is this formatted? If it is, we may remove this comma to make it as one line. Some other places are like this too.